public/assets/base.css

```diff
@@ -44,3 +44,8 @@
   font-weight: 400;
   font-style: normal;
 }
+
+.splash-screen {
+  /* lightModeBrown.11  */
+  background-color: #12100F !important;
+}
```

Comment:
Can we set this via panda globalCss?

Reply:
hm, yeah, makes sense. will look into it